internal/stack/resources.go

```diff
@@ -122,10 +122,8 @@ var (
 			Content: staticSource.Template("_static/elastic-agent.env.tmpl"),
 		},
 		&resource.File{
-			Path:         "logstash_startup.sh",
-			CreateParent: true,
-			Content:      staticSource.Template("_static/logstash_startup.sh"),
-			Mode:         resource.FileMode(0755),
+			Path:    "Dockerfile.logstash",
+			Content: staticSource.File("_static/Dockerfile.logstash"),
 		},
```

Comment:
There is also another reference to this file in serverlessresources.go

Trying to creaste a serverless Elastic stack raised this error:

```
2024/02/15 18:41:48  INFO Starting local services
Error: booting up the stack failed: failed to start local services: could not initialize compose files for local services: there where errors: template: pattern matches no files: `_static/logstash_startup.sh`
```
https://github.com/elastic/elastic-package/blob/b2ee26459d1b91f07de567f0db126174d5328b34/internal/stack/serverlessresources.go#L38